Dynamic-Programming/ContinuousSubarraySum/README.md

```diff
@@ -0,0 +1,112 @@
+## Continous Subarray Sum
+
+<p>
+<img align="center" width="800px" alt="Question Screenshot" src="./../../Images/ContinuousSubarraySum/question.png">
+</p>
+
+---
+
+###  Motivation
+We are given an array of Integers `nums` and `K`. we have to find a continuous subset of the array of `length>=2` whose sum is of the form `N*K` where `N` is any integer. 
+
+
+### Solution-1 : Naive Way
+
+The naive solution involves traversing all the subset of `nums` whose `length>=2` and checking for `nums[i:j]%K==0` where `i<j` and `nums[i:j]= nums[i]+nums[i+1]+...+nums[j-1]+nums[j]`. While traversing the subsets we explicitily need handle the case `K=0` because `nums[i:j]%K` will throw `DivideByZeroException`.
+
+
+#### Algorithm
+1. Create all the subsets of `nums` array whole `length>=2`.
+2. Traverse the subsets and check if their sum is a multiple of `K`
+3. While traversing the subsets check whether `K=0` which requires sum of the elements in the subset be equal to zero.
+   
+<p>
+<img align="center" width="800px" alt="Naive Algorithm" src="./../../Images/ContinuousSubarraySum/ContinuousSubArraySum-NaiveAlgo.png">
+</p>
+
+This solution will throw `Time Limit Exceeded` error when run on leet code.
+
+
+
+### Solution-2 : Dynamic Programming
```

Comment:
This is not exactly a dynamic programming solution. We are just doing some preprocessing to avoid repeated computation. Any DP solution has two main properties:

1. It can be broken down into sub-problems and 
2. We can make use of optimal solutions for sub-problems for solving the main problem.

This solution doesn't fit the bill as far as these conditions are concerned. It would be better to name it as `Accumulated sum pre-processing` or something. At the end of the day, we still check each and every subset for the condition and I don't think there is a recursive element to this problem.